core/src/main/scala/org/broadinstitute/dsde/rawls/deltalayer/DeltaLayer.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.rawls.deltalayer
+
+import java.util.UUID
+
+object DeltaLayer {
+  def generateDatasetName(datasetReferenceId: UUID) = {
+    "deltalayer_" + datasetReferenceId.toString.replace('-', '_')
+  }
+}
```

Comment:
moved out of `SnapshotService` so it can be easily referenced from multiple places

Reply:
hm, I can't speak authoritatively on conventions, but I have renamed the model file from DeltaLayer.scala to DeltaLayerModel.scala, which matches other model files. Good call.